src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.seachBar {
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.seachBar_input_big {
+  height: 70px;
+  padding-left: 62px;
+  line-height: 62px;
+  font-size: 16px;
+}
+
+.seachBar_input_small {
+  height: 42px;
+  padding-left: 33px;
+  line-height: 42px;
+  font-size: 14px;
+}
+
+.seachBar_input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: bold;
+}
+
+.seachBar_input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.seachBar_input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.seachBar_input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.seachBar:before {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  content: "";
+  position: absolute;
+  display: block;
```

Comment:
read about sorting rules. It eases reading code a lot 
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/